manifests/charts/istio-control/istio-discovery/templates/deployment.yaml

```diff
@@ -156,6 +156,10 @@ spec:
             value: "{{ $.Values.global.externalIstiod | default "false" }}"
           - name: CENTRAL_ISTIOD
             value: "{{ $.Values.global.centralIstiod | default "false" }}"
+{{- if .Values.telemetry.v2.enabled }}
```

Comment:
very nit, but I would prefer to default to enabled, and disable it if telemetry is off. that way for the common case the deployment is smaller - optimizing for readability and ability to run istiod without 100s of options, not for the etcd storage size

Reply:
Makes sense. Reversed the condition.